### ISSUE TITLE
Is resnet56 getting overfit to achieve 93% accuracy? Could you please provide accuracy and loss graph for resnet56?

### ISSUE THREAD

**Issue by tomshalini** [akamaster/pytorch_resnet_cifar10#26]
When I trained resnet 56 on cifar10 and plotted the graphs. It's getting overfit as shown below.

![image](https://user-images.githubusercontent.com/59674741/107880237-ded10480-6edd-11eb-880f-450caf1f2f3f.png)

**Comment by akamaster** [comment]
I have about the same plots. The phenomenon you are observing is common in deep nets, which actually is a topic of interest. See for example https://openreview.net/pdf?id=Sy8gdB9xx
